suite/tests/api/ir_x86.c

```diff
@@ -2168,6 +2168,97 @@ test_reg_exact_reads(void *dc)
     instr_destroy(dc, instr);
 }
 
+static void
+test_re_relativization_disp32_opc16(void *dcontext, byte opc1, byte opc2)
+{
+    byte buf_dec_enc[] = { opc1, opc2,
+                           /* disp32 of 0 which targets the next PC. */
+                           0x00, 0x00, 0x00, 0x00,
+                           /* We encode here. */ 0x90, 0x90, 0x90, 0x90, 0x90, 0x90 };
```

Comment:
I'm guessing this line-joining is due to clang-format? Consider adding whitespace after the "We encode here." comment so that clang-format decides to wrap like the previous line is wrapped.

(alternatively we could use a line-comment, but that's against the existing style)

Reply:
Newline successfully added and preserved by clang-format.